spec/features/shelters/show_spec.rb

```diff
@@ -77,3 +77,51 @@
     end
   end
 end
+
+describe 'As a visitor' do
+  describe 'When I visit a shelters show page' do
+    it 'should show a list of reviews for that shelter' do
+      shelter1 = Shelter.create!(name: 'Dogs and Cats',
```

Comment:
I think we should add an edge case shelter2 plus a review for shelter2 in this test to make sure that only the reviews for shelter1 are displayed on the shelter1 display page.